workshops/static/online_country.js

```diff
@@ -0,0 +1,42 @@
+var prev_venue,
+    prev_address,
+    prev_latitude,
+    prev_longitude,
+    online_selected_before = false;
+
+var id_country = '#id_country, #id_event-country',
+    id_venue = '#id_venue, #id_event-venue',
+    id_address = '#id_address, #id_event-address',
+    id_latitude = '#id_latitude, #id_event-latitude',
+    id_longitude = '#id_longitude, #id_event-longitude';
+
+// disable latitude, longitude inputs if online country selected
+$(id_country).change(function(e) {
+  var new_country = $(e.target).val();
+
+  if (new_country == 'W3') {
+    prev_venue = $(id_venue).val();
+    prev_address = $(id_address).val();
+    prev_latitude = $(id_latitude).val();
+    prev_longitude = $(id_longitude).val();
+    $(id_venue).val('Internet');
+    $(id_address).val('Internet');
+    $(id_latitude).val('-48.876667');
+    $(id_longitude).val('-123.393333');
```

Comment:
This approach assumes all oneline-event updates will go through an AMY UI using this JavaScript and that admins won't tweak the magic values after the JavaScript here populates the field values.  I'd rather make `Event.country` the canonical source of online-ness, and have `Event.save()` *force* all the associated fields to their magic online values, because I have more faith in `Event.save()` being in the update path ;).